Traceback (most recent call last):
  File "C:\MyDocuments\Disertatie\segments\wandb_visualization\overlap\wrapper_test_and_train.py", line 80, in <module>
    train_models_and_save(n_members,data,test_size,train_group, epochs, batch_size, learning_rate,optimizer,hidden_layer_size,patience,monitor)
  File "C:\MyDocuments\Disertatie\segments\wandb_visualization\overlap\test_and_train.py", line 17, in train_models_and_save
    x_train_cnn = x_train.reshape(3883, 8, 1, 1)
ValueError: cannot reshape array of size 41288 into shape (3883,8,1,1)